src/panels/lovelace/hui-view.js

```diff
@@ -133,7 +135,17 @@ class HUIView extends PolymerElement {
     const elements = config.cards.map((cardConfig) => {
       const element = createCardElement(cardConfig);
       element.hass = this.hass;
-      return element;
+      if (!this.editMode) {
+        return element;
+      }
+
+      const wrapper = document.createElement("hui-card-options");
```

Comment:
We're having a problem now which is that cards won't be updated with new `hass`, because `elements` only contains the wrappers. I think that we need to maintain 2 lists: one that needs to receive new `hass`, one that needs to be appended to the DOM.